Sources/_FoundationCShims/include/platform_shims.h

```diff
@@ -68,4 +68,14 @@ typedef enum {
 INTERNAL const char * _Nonnull _platform_shims_kOSThermalNotificationPressureLevelName();
 #endif
 
+// Define clock_gettime shims for each clock type so that we can use them in Swift
+// even if clock id macros can't be imported through ClangImporter.
+
+#if !defined(_WIN32)
```

Comment:
Could we use `TARGET_OS_WINDOWS` here to try to be consistent with our platform checks (I know some places might not, I plan to come and fix up a few but it'd be great to standardize on the target conditionals with new additions)